src/tests/test_parse.py

```diff
@@ -844,9 +844,9 @@ def test_escape_string_slicing_notation(query, expected_escaped, expected_found)
             "",
         ),
         (
-            "select * from authors",
             "select * from authors",
             "",
+            "select * from authors",
```

Comment:
what exactly changed here?

Reply:
I changed the logic for `split_args_and_sql` to split any line that includes a SQL keyword. Previously it would only split the line if it included json operators `->` or `-->`. For this test case, it was previously NOT split. But with the changes I made, it now should be split.